imports/plugins/included/surcharges/server/no-meteor/util/getSurchargeMessageInShopLanguage.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name getSurchargeMessageInShopLanguage
+ * @summary Returns translated surcharge message
+ * @param {String} language Language to filter items by
+ * @param {Array} messagesByLanguage Surcharge messages by language array
+ * @return {String} Translated surcharge message, or first surcharge message if translation in
+ *  provided language is not available.
+ */
+export default function getSurchargeMessageInShopLanguage(language, messagesByLanguage) {
+  const translatedMessage = messagesByLanguage.find((message) => message.language === language);
+  if (translatedMessage) {
+    return translatedMessage.content;
+  }
+  return messagesByLanguage[0].content || "";
```

Comment:
This seems wrong since we don't even know what language will be the first one in the array. I would just return null (`return translatedMessage && translatedMessage.content || null;`)

Reply:
That's fair. I modeled this after how we do it for the navigation, should we update it there as well? Or is this a different circumstance where we definitely need something to show for navigation?

Reply:
Navigation seems less clear-cut than this, but I'd lean toward that returning `null`, too.

I also think it would be much more straightforward and versatile to move the `language` argument to be directly on the `message` field. Then you don't need to attach it to the surcharge object in the query resolver.

Reply:
To be clear, I mean you'd query like this instead:

```
query {
  getSurcharge(shopId: "shopId-GOES-HERE", surchargeId: "surchargeId-GOES-HERE") {
    _id
    shopId
    message(language: "en")
  }
}
```